FeedStoreChallenge/Core Data/CoreDataStack.swift

```
@@ -0,0 +1,15 @@
+
+import Foundation
+import CoreData
+
+
+public protocol CoreDataStack {
+    
+    var managedContext: NSManagedObjectContext { get }
+    
+    init(storeURL: URL)
+
+    func deleteAll(of entityName: String, context: NSManagedObjectContext) -> Error?
+    func saveContext(context: NSManagedObjectContext) -> Error?
```

It'd be best to throw errors instead of returning optional `Error?` - it's more idiomatic in Swift.